decidim-admin/app/commands/decidim/admin/destroy_static_page_topic.rb

```diff
@@ -1,40 +0,0 @@
-# frozen_string_literal: true
-
-module Decidim
-  module Admin
-    # This command deals with destroying a StaticPageTopic from the admin panel.
-    class DestroyStaticPageTopic < Decidim::Command
-      # Public: Initializes the command.
-      #
-      # page - The StaticPageTopic to be destroyed.
-      def initialize(_topic, current_user)
-        @topic = page
```

Comment:
Maybe it's too early for me, but where does `page` come? How does this work?

Reply:
Don't know exactly. but i am sure is waay to early. The question is on a file that i am removing :)

Reply:
Yes, I know that it's a removal 😄 

The thing is that is a mini WTF, like where `page` come? Why does this even work??

Reply:
As we're removing this weirdness, we can let it sly and go on with the merge here